src/elements/tooltip/tooltip.ssr.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { assert } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+
+import { ssrHydratedFixture } from '../core/testing/private.js';
+
+import { SbbTooltipElement } from './tooltip.component.js';
+
+describe(`sbb-tooltip ssr`, () => {
+  let root: SbbTooltipElement;
+
+  beforeEach(async () => {
+    root = await ssrHydratedFixture(html`<sbb-tooltip>Test</sbb-tooltip>`, {
```

Comment:
I would prefer something more 'real', like:

```suggestion
    root = await ssrHydratedFixture(html`
      <sbb-button id="test-id">Button</sbb-button>
      <sbb-tooltip trigger="test-id">Test</sbb-tooltip>
`, {
```